backend/lib/models/Post.js

```diff
@@ -69,6 +69,16 @@ const postSchema = new Schema(
       ref: "User",
       type: [ObjectId],
     },
+    views: {
+      default: [],
```

Comment:
I don't know if we'll ever need to see which user viewed the post, if the only information required is to display the count of views instead of specific users viewed the post, we may use a number instead?